### ISSUE TITLE
Unable to run sample project on Linux

### ISSUE THREAD

**Issue by ccpplinux** [haxenme/nme#701]
When I am trying to run the sample project on my Manjaro Linux laptop with command "nme test linux" then I am getting following error message:

/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:72: characters 3-25 : error: nme.Vector.T should be iterator.T
/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:72: characters 3-25 : ... have: { next: (...) -> nme.Vector.T }
/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:72: characters 3-25 : ... want: { next: (...) -> iterator.T }
/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:233: characters 3-14 : error: nme.Vector.T should be toArray.T
/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:233: characters 3-14 : ... have: nme._Vector.VectorData<nme.Vector.T>
/home/ccpplinux/haxelib/nme/6,1,1/src/nme/Vector.hx:233: characters 3-14 : ... want: Array<toArray.T>

Similarly in case of command "nme test html5" I am getting following error:

Error: Library source path "/home/ccpplinux/haxelib/nme/6,1,1/ndll/Emscripten/Nme.js" does not exist +({ registerPrim => null, name => Nme, path => /home/ccpplinux/haxelib/nme/6,1,1, haxelibName => , isStatic => false, importStatic => , noCopy => false })

Can you please suggest fix for these problems.

**Comment by hughsando** [comment]
The haxelib version is not currently the most up-to-date version.
You can try one of the development builds at http://nmehost.com/nme/ by downloading the zip and using "haxelib install xxxxx.zip"
Alternatively, you can build from source, which involves downloading from github and compiling the dll by running "neko build.n" in the project directory

**Comment by ccpplinux** [comment]
Thanks for giving me hint. I will try and then let you know.

**Comment by ccpplinux** [comment]
Yes now it is working. I am able to build for the target neko and linux and run the program. It is really exciting. But when I am building for the target html5 then I am getting following error:

Error: Library source path "/home/ccpplinux/haxelib/nme/6,1,33/ndll/Emscripten/Nme.js" does not exist +({ registerPrim => null, name => Nme, path => /home/ccpplinux/haxelib/nme/6,1,33, haxelibName => , isStatic => false, importStatic => , noCopy => false })

Similarly when I am building for target android then I am getting following error message:

haxelib run hxcpp Build.xml haxe -DHXCPP_ARMV7="1" -Dandroid="1" -Dgradle="1" -Dhaxe="4.2.4" -Dhaxe3="1" -Dhaxe4="1" -Dhaxe_ver="4.204" -Dhxcpp_api_level="400" -Dhxcpp_smart_strings="1" -Dlime_legacy="1" -Dmobile="1" -Dnme="6.1.33" -Dnme6="1" -Dnme_api_level="611" -Dnme_install_tool="1" -Dnme_static="1" -Dnme_ver="6.1.33" -Dopenfl="3.5.0" -Dopenfl_legacy="1" -Dsource_header="Generated by Haxe 4.2.4" -Dstatic="1" -Dtarget.name="cpp" -Dtarget.static="true" -Dtarget.sys="true" -Dtarget.threaded="true" -Dtarget.unicode="true" -Dtarget.utf16="true" -Dtoolkit="1" -Dutf16="1" -I"Export/android/haxe/" -I"/home/ccpplinux/haxelib/nme/6,1,33/src/haxe/compat/" -I"/home/ccpplinux/haxelib/nme/6,1,33/src/" -I"Source/" -I"/home/ccpplinux/haxelib/nme/6,1,33/src/" -I"" -I"/home/ccpplinux/Nextcloud/Haxe/haxe_latest/std/cpp/_std/" -I"/home/ccpplinux/Nextcloud/Haxe/haxe_latest/std/"
Error: Could not detect Android API platforms in "/home/ccpplinux/Android/Sdk/ndk/23.1.7779620/package.xml/platforms"
Error: Build failed

Please note that on my laptop android setup is working fine as I am able to generate apk file using Flutter and GioUI. So it should also work with haxenme but it is not working.

Can you give me hint for these two targets that is html5 and android?
